crates/typst/src/text/raw.rs

```diff
@@ -852,6 +862,13 @@ pub static RAW_THEME: Lazy<synt::Theme> = Lazy::new(|| synt::Theme {
     ],
 });
 
+/// The theme used to disable syntax highlighting
+pub static RAW_THEME_NONE: Lazy<synt::Theme> = Lazy::new(|| synt::Theme {
```

Comment:
Maybe instead having a "nothing theme", we should rather just not highlight in the first place?

Reply:
Agree. In a big book there will be many instances of raw elements, so this is not a case of premature optimization.

Reply:
Agree. I have fixed it. Now it will early exit when `raw.theme` is `none`.